src/marqo/tensor_search/tensor_search.py

```diff
@@ -583,8 +597,11 @@ def add_documents(config: Config, add_docs_params: AddDocsParams):
                                         f"Could not find image found at `{field_content}`. \n"
                                         f"Reason: {str(image_repo[field_content])}"
                                     )
+                            elif isinstance(field_content, str):
+                                image_data = text_chunk_prefix + field_content     # Add prefix to URL if it's to be treated as-is.
                             else:
-                                image_data = field_content
+                                image_data = field_content      # If it's actual image data, just pass it through.
```

Comment:
I feel like this might be unreachable. But let's leave it for later

Reply:
Agreed, I just left it in to not break anything, just in case. It should be cleared up in v2 anyway.